docs/examples/Reinforcement_Learning_Sensor_Manager.py

```diff
@@ -0,0 +1,1002 @@
+# Full explanations of the OSPA
+# and SIAP metrics can be found in the Metrics Example.
+
+from stonesoup.metricgenerator.ospametric import OSPAMetric
+ospa_generatorA = OSPAMetric(c=40, p=1,
+                             generator_name='ReinforcementSensorManager',
+                             tracks_key='tracksA',
+                             truths_key='truths')
+
+ospa_generatorB = OSPAMetric(c=40, p=1,
+                             generator_name='BruteForceSensorManager',
+                             tracks_key='tracksB',
+                             truths_key='truths')
+
+from stonesoup.metricgenerator.tracktotruthmetrics import SIAPMetrics
+from stonesoup.measures import Euclidean
+siap_generatorA = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                              velocity_measure=Euclidean((1, 3)),
+                              generator_name='Reinforcement',
+                              tracks_key='tracksA',
+                              truths_key='truths')
+
+siap_generatorB = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                              velocity_measure=Euclidean((1, 3)),
+                              generator_name='BruteForceSensorManager',
+                              tracks_key='tracksB',
+                              truths_key='truths')
+
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+associator = TrackToTruth(association_threshold=30)
+
+from stonesoup.metricgenerator.uncertaintymetric import SumofCovarianceNormsMetric
+uncertainty_generatorA = SumofCovarianceNormsMetric(generator_name='ReinforcementSensorManager',
+                                                    tracks_key='tracksA')
+
+uncertainty_generatorB = SumofCovarianceNormsMetric(generator_name='BruteForceSensorManager',
+                                                    tracks_key='tracksB')
+
+# %%
+# Generate a metrics manager.
+
+from stonesoup.metricgenerator.manager import MultiManager
+
+metric_manager = MultiManager([ospa_generatorA,
+                               ospa_generatorB,
+                               siap_generatorA,
+                               siap_generatorB,
+                               uncertainty_generatorA,
+                               uncertainty_generatorB],
+                               associator=associator)
+
+# %%
+# For each time step, data is added to the metric manager on truths and tracks. The metrics themselves can then be
+# generated from the metric manager.
+
+metric_manager.add_data({'truths': truths, 'tracksA': tracksA, 'tracksB': tracksB})
+
+metrics = metric_manager.generate_metrics()
+
+# %%
+# OSPA metric
+# -----------
+# First we look at the OSPA metric. This is plotted over time for each sensor manager method.
+
+from stonesoup.plotter import MetricPlotter
+
+fig = MetricPlotter()
+fig.plot_metrics(metrics, metric_names=['OSPA distances'])
+
+# %%
```

Comment:
Obvious question: can you explain why the reinforcement-learning sensor management is sometimes better than the 'optimal' brute-force manager?

Reply:
I'm actually not totally sure, I think it's just that at some points, the brute force algorithm will pick the target that is most uncertain, and the reinforcement picks another target that happens to reduce OSPA distance more.

Reply:
I think you're right -- but I also think it would be worth mentioning that in the notebook as the naive assumption that the brute force algorithm must obviously always be best may be difficult to unstick.